$NetBSD: patch-doc_meson.build,v 1.1 2021/05/07 07:18:08 wiz Exp $

meson recognizes that it needs rsync. This target is unused,
remove it from pkgsrc build to avoid rsync dependency.

--- doc/meson.build.orig	2021-02-16 12:56:14.000000000 +0000
+++ doc/meson.build
@@ -23,19 +23,6 @@ if get_option('html_manual')
     install_dir: join_paths(get_option('datadir'), 'doc', meson.project_name()),
   )
 
-  custom_target(
-    'upload',
-    input: sphinx_output,
-    output: 'upload',
-    build_always_stale: true,
-    command: [
-      'rsync', '-vpruz', '--delete', meson.current_build_dir() + '/',
-      'www.musicpd.org:/var/www/mpd/doc/',
-      '--chmod=Dug+rwx,Do+rx,Fug+rw,Fo+r',
-      '--include=html', '--include=html/**',
-      '--exclude=*',
-    ],
-  )
 endif
 
 if get_option('manpages')
